examples/generative/corrdiff/datasets/img_utils.py

```diff
@@ -17,6 +17,7 @@
 
 import numpy as np
 import torch
+import warnings
 
 
 def reshape_fields(
```

Comment:
It's a bit confusing to have the position encoding in a function called "reshape fields". Perhaps this functions does too much and should be split into two.